src/cli/commands/test/sarif-output.ts

```diff
@@ -1,4 +1,5 @@
 import * as sarif from 'sarif';
+import upperFirst = require('lodash/upperFirst');
```

Comment:
You can't use import?
```suggestion
import upperFirst from 'lodash/upperFirst';
```

Reply:
This solution would not help because `upperFirst` was exported as `export = upperFirst;`
and in order to allow it we need to change `tsconfig.json` **compilerOptions** and add a new property
`"esModuleInterop": true,`

Reply:
Was a request in the original PR. Considering the two comments on this now, changed 👍